vortex-array/src/compute/list.rs

```diff
@@ -357,14 +334,19 @@ mod tests {
     #[case(
         null_strings(vec![vec![], vec![None, None], vec![None, None, None]]),
         None,
-        bool_array(vec![false, true, true], None)
+        bool_array(vec![false, true, true], Some(vec![false, false, false]))
     )]
     // Case 7: list(utf8?) with empty + NULL elements and search scalar
     #[case(
         null_strings(vec![vec![], vec![None, None], vec![None, None, None]]),
         Some("a"),
         bool_array(vec![false, false, false], None)
     )]
+    #[case(
```

Comment:
Missing Cast 8: what's the intention of this test case, it should be clear that this isn't already tested above